doc/source/groupby.rst

```diff
@@ -62,16 +62,30 @@ See the :ref:`cookbook<cookbook.grouping>` for some advanced strategies.
 Splitting an object into groups
 -------------------------------
 
-pandas objects can be split on any of their axes. The abstract definition of
+Pandas objects can be split on any of their axes. The abstract definition of
 grouping is to provide a mapping of labels to group names. To create a GroupBy
-object (more on what the GroupBy object is later), you may do the following:
+object, see below (more on what the GroupBy object is later). A
+groupby can be applied in the following ways to a pandas object:
+
+* grouped = obj.groupby(key)
+* grouped = obj.groupby(key, axis='columns')
+* grouped = obj.groupby([key1, key2])
 
 .. code-block:: python
 
-   # default is axis=0
-   >>> grouped = obj.groupby(key)
-   >>> grouped = obj.groupby(key, axis=1)
-   >>> grouped = obj.groupby([key1, key2])
+    df = pd.DataFrame(
+        [('bird', 'Falconiformes', 389.0),
+         ('bird', 'Psittaciformes', 24.0),
+         ('mammal', 'Carnivora', 80.2),
+         ('mammal', 'Primates', np.nan),
+         ('mammal', 'Carnivora', 58)],
+        index=['falcon', 'parrot', 'lion', 'monkey', 'leopard'],
+        columns=('class', 'order', 'max_speed')
+    )
+
+    grouped = df.groupby('class')
```

Comment:
Besides this needs to be an `.. ipython:: python` block (otherwise the `>>>` and the output needs to be shown, please show the content of `df` after defining it, so it's easier to understand what the groupby is doing, and leave the comment saying that the default axis is index

Reply:
I have changed codeblock to```.. ipython:: python```, printed out the content of ```df```, and left the comment about default axis. (see c30f4567b60298836f867515f0f761da8807a3f0)